alot/account.py

```diff
@@ -20,6 +20,7 @@ class StoreMailError(Exception):
 
 
 class Account(object):
+
```

Comment:
yes, I meant the other docstring. I don't really mind, just wanted to make sure we stick to one convention.